modules/audio_device/win/core_audio_base_win.cc

```diff
@@ -388,15 +388,27 @@ bool CoreAudioBase::Init() {
     }
   }
 
+  // RingRTC change to get the mix_format keep it in scope for multi-channel.
+  WAVEFORMATPCMEX mix_format;
+  HRESULT res = core_audio_utility::GetSharedModeMixFormat(
+      audio_client.Get(), &mix_format);
+  if (FAILED(res)) {
+    return false;
+  }
+
+  RTC_LOG(LS_WARNING) << (IsInput() ? "input " : "output ")
+                      << "mix_format: "
+                      << core_audio_utility::WaveFormatToString(&mix_format);
```

Comment:
Nitpick: this should probably be `LS_INFO` since it's on the happy path.

Reply:
I would like to see it in logs for a while, to get a reference for other audio issues on Windows, that is why I have it at warning level for now.